src/lib/ui/ProNotebookFocus/ProNotebookFocusDetails.svelte

```diff
@@ -4,17 +4,22 @@
 		DeleteNotebookFocusByIdDocument,
 		GetNotebookFocusByIdDocument,
 	} from '$lib/graphql/_gen/typed-document-nodes';
+
+	import { UpdateTargetStatusDocument } from '$lib/graphql/_gen/typed-document-nodes';
+
+	import type { UpdateTargetStatusMutation } from '$lib/graphql/_gen/typed-document-nodes';
 	import { openComponent } from '$lib/stores';
 	import { trackEvent } from '$lib/tracking/matomo';
-	import { Accordion, Accordions, Button, Card } from '$lib/ui/base';
+	import { Accordion, Accordions, Button, Card, Select } from '$lib/ui/base';
 	import Dialog from '$lib/ui/Dialog.svelte';
 	import { displayFullName } from '$lib/ui/format';
 	import { Text } from '$lib/ui/utils';
-	import { mutation, operationStore, query } from '@urql/svelte';
+	import { mutation, OperationStore, operationStore, query } from '@urql/svelte';
 	import { ProNotebookActionList } from '../ProNotebookAction';
 	import ProNotebookCreatorView from '../ProNotebookCreator/ProNotebookCreatorView.svelte';
 	import ProNotebookTargetCreate from '../ProNotebookTarget/ProNotebookTargetCreate.svelte';
 	import ProNotebookFocusUpdate from './ProNotebookFocusUpdate.svelte';
+	import { ActionStatus } from '$lib/enums';
```

Comment:
question(non-blocking): Je sais que ça vient d'une précédente PR (#650), mais j'avais l'impression que les Enums étaient découragés, au profit des union types. J'ai trouvé cet article qui va dans ce sens, mais évidemment ça n'a pas vocation à être définitif (surtout que l'auteur se moque lui-même de sa non-objectivité =)) : https://blog.bam.tech/developer-news/should-you-use-enums-or-union-types-in-typescript
C'est pas bloquant, mais ça vaut ptet le coup d'en discuter pour plus tard ?